envoy/http/header_validator.h

```diff
@@ -106,8 +106,9 @@ class HeaderValidatorStats {
 public:
   virtual ~HeaderValidatorStats() = default;
 
-  virtual void incDroppedHeadersWithUnderscores() PURE;
-  virtual void incRequestsRejectedWithUnderscoresInHeaders() PURE;
+  // These can't be PURE for now, otherwise coverage in non UHV builds is unhappy.
```

Comment:
optional: would it make more sense to leave these pure, remove the guard around the implementation, and add guards around invocation?
If you want to leave as-is I think a TODO to remove base class impl a part of the UHV bug would be good.

Reply:
I agree. I realized I could fix coverage by using these methods in the ifdefed code.